corehq/apps/users/views/mobile/users.py

```diff
@@ -302,21 +312,11 @@ def page_context(self):
             'data_fields_form': self.form_user_update.custom_data.form,
             'can_use_inbound_sms': domain_has_privilege(self.domain, privileges.INBOUND_SMS),
             'show_deactivate_after_date': self.form_user_update.user_form.show_deactivate_after_date,
-            'can_create_groups': (
```

Comment:
nit: I do prefer the way this was written before the refactor. I would only separate these into variables like you did if these were used more than in one place to be DRY. But since they are only used once, pulling them into a variable makes it less readable (one has to lookup the variables while reading which is an overhead).

Reply:
It makes sense. I have no strong feelings about this. The only reason I did this refactor was because the linter complained about the line (306) ending on a binary operator.